code/modules/clothing/suits/cloaks.dm

```diff
@@ -6,7 +6,7 @@
 	icon = 'icons/obj/clothing/cloaks.dmi'
 	icon_state = "qmcloak"
 	w_class = 2
-	slot_flags = SLOT_BACK
+	slot_flags = SLOT_OCLOTHING
```

Comment:
You shouldn't need to define the slot_flags at all if its a suit, it'll do it automatically unless it's overridden with something else as far as I know.